neptune/new/run.py

```diff
@@ -56,6 +56,51 @@
 
 
 class Run(AbstractContextManager):
+    """A Run in Neptune is a representation of all metadata that you log to Neptune.
+
+    Beginning when you start a tracked run with `neptune.init()` and ending when the script finishes
+    or when you explicitly stop the experiment with `.stop()`.
+
+    You can log many ML metadata types, including:  TODO
```

Comment:
Why TODO?

Reply:
My bad, I forgot to remove it after I've fixed below list.